tests/test_content_negotiation.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Flask--Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from werkzeug.datastructures import MIMEAccept
+from werkzeug.http import parse_accept_header
+
+from flask_resources.content_negotiation import ContentNegotiator
+
+
+# Test content negotiation by Accept header
+# NOTE: By scoping down we remove the need to check for HTTP method
+def test_choose_provided_and_accepted_mimetype():
+    # Should choose mimetype that is accepted by client and served by server
+    server_mimetypes = ['application/json', 'application/marcxml+xml']
+    client_mimetypes = parse_accept_header(
+        'text/plain,application/json,*/*',
+        MIMEAccept
+    )
+
+    mime_type = ContentNegotiator.match_by_accept(
```

Comment:
comment: style-wise I personally prefer shorter test code like this

```python
assert 'application/json' == ContentNegotiator.match_by_accept(...)
```

Reply:
Sure. Makes sense here.